cmd/kube-proxy/app/BUILD

```diff
@@ -64,7 +64,6 @@ go_library(
         "//pkg/apis/core:go_default_library",
         "//pkg/client/clientset_generated/internalclientset:go_default_library",
         "//pkg/client/informers/informers_generated/internalversion:go_default_library",
-        "//pkg/kubectl/cmd/util:go_default_library",
```

Comment:
Please remove this line:

https://github.com/kubernetes/kubernetes/blob/835afe683f7e6b1011142beab1ce65eba0a5e8ff/build/visible_to/BUILD#L224

To prevent this dependency from reoccuring.

Reply:
Thanks for remind